app-backend/db/src/main/scala/SceneDao.scala

```diff
@@ -309,6 +312,18 @@ object SceneDao
     }
   }
 
+  def getSentinelMetadata(
+      metadataUrl: String): ConnectionIO[(Array[Byte], ObjectMetadata)] = {
```

Comment:
Could make sense to LiftIO here again -- `F.pure` is useful for putting pure values into some context, while here we have a side-effect that's getting `pure`'d up